data/pumi/geom/Kova.dmg

```diff
@@ -0,0 +1,95 @@
+1 8 18 12
```

Comment:
We prefer to keep only simple example meshes in `data/` can you move the content of `data/pumi` to the external [mfem/data repo](https://github.com/mfem/data)  instead and reference them as in https://github.com/mfem/mfem/blob/master/miniapps/electromagnetics/joule.cpp#L89-L93?

Reply:
Done. PUMI examples' `data` is moved to the external [mfem/data/pumi repo](https://github.com/mfem/data/tree/master/pumi)